packages/office-addin-debugging/src/start.ts

```diff
@@ -60,7 +61,17 @@ export async function isPackagerRunning(statusUrl: string): Promise<boolean> {
 
 export async function isUrlAvailable(url: string): Promise<boolean> {
     try {
-        await fetch.default(url);
+        const serverOptions = await devCerts.getHttpsServerOptions();
+        const options: any = {
```

Comment:
Please don't use "any". Do you even need to specify a type? Isn't it inferred from the assigned value?

Reply:
I couldn't get this to compile unlesss I use any